jwst/pipeline/calwebb_image2.py

```diff
@@ -108,8 +109,8 @@ def process_exposure_product(
             input = datamodels.open(science)
 
         # Record ASN pool and table names in output
-        input.meta.asn.pool_name = pool_name
-        input.meta.asn.table_name = asn_file
+        input.meta.asn.pool_name = op.basename(pool_name)
```

Comment:
I think `pool_name` is always pulled from inside the association, so it never has a path associated with it.  Is that correct?  If it does have a path, then we should use `basename()` on all the other places in the pipeline we populate `pool_name`.